Android/Java/app/src/main/java/com/ironsource/ironsourcesdkdemo/DemoActivity.java

```diff
@@ -119,51 +134,54 @@ public void showRewardedVideoButtonTapped(View view) {
 
     public void loadInterstitialButtonTapped(View view) {
         // This will load an Interstitial ad
-
+        mInterstitialAd = new LevelPlayInterstitialAd(INTERSTITIAL_AD_UNIT_ID);
+        mInterstitialAd.setListener(new DemoInterstitialAdListener(this));
+        mInterstitialAd.loadAd();
         log("loadInterstitial");
-        IronSource.loadInterstitial();
     }
 
     public void showInterstitialButtonTapped(View view) {
         // It is advised to make sure there is available ad before attempting to show an ad
-        if (IronSource.isInterstitialReady()) {
+        if (mInterstitialAd.isAdReady()) {
             // This will present the Interstitial.
             // Unlike Rewarded Videos there are no placements.
 
             log("showInterstitial");
-            IronSource.showInterstitial();
+            mInterstitialAd.showAd(this);
         } else {
             // load a new ad before calling show
         }
     }
 
     public void loadBannerButtonTapped(View view) {
-        // call IronSource.destroyBanner() before loading a new banner
-        if (bannerParentLayout != null) {
-            this.destroyBanner();
+        // call destroy() before loading a new banner
+        if (mBannerAd != null) {
+            destroyBanner();
```

Comment:
I think we need to guide them to call the load function only once on the object,  and in this way it's not clear. 
Moreover, it appears to require creating a new object for each load. I suggest removing the destroy logic and allowing the load function to be called on the same object with a relevant comment. For example: `The banner is refreshed automatically after the first load, so there's no need to call it again on the same object`

Reply:
this is a change in logic from the current demo, the decision was not to change any logic. The current integration is to load every time regardless of whether the ad is loaded or not, not just for banner but for interstitial as well
@ronala  what do you think?

Reply:
Hi, this is the text I had in mind (for Load API): 

Load a banner ad. If the "refresh" option is enabled in the LevelPlay dashboard settings, the banner will automatically refresh at the specified interval. Otherwise, the banner will remain static until manually destroyed.